pkg/endpoint/http.go

```diff
@@ -0,0 +1,66 @@
+package endpoint
+
+import (
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+var (
+	etcdVersion = []byte(`{"etcdserver":"3.5.0","etcdcluster":"3.5.0"}`)
```

Comment:
Would be slick if this version could be lifted from etcd server code somewhere but then it would likely have to be injected at build time :man_shrugging:

Reply:
Yeah, I considered getting more clever with it but ended up just hardcoding it as suggested at https://github.com/k3s-io/kine/issues/30#issuecomment-622977646